app/app.js

```diff
@@ -175,3 +175,5 @@ app.post('/sloth-check', (req, res) => {
 app.listen(app.get('port'), function() {
     console.log('App is running, server is listening on port', app.get('port'))
 })
+
+module.exports.parseTensorFlowResult = parseTensorFlowResult
```

Comment:
This works for the purposes of the test you wrote, but what would be more testable is to separate the logic out into its own file and `require` it into the app file. That way, you can also test the parsing logic without running the whole server.

See the tests in my [time-tiles](https://github.com/andrewvora/time-tiles/tree/master/app/users) project for an example of what I mean